src/sentry/testutils/cases.py

```diff
@@ -3164,3 +3168,230 @@ def setUp(self):
 class IntegratedApiTestCase(BaseTestCase):
     def should_call_api_without_proxying(self) -> bool:
         return not IntegrationProxyClient.determine_whether_should_proxy_to_control()
+
+
+class SpanTestCase(BaseTestCase):
```

Comment:
I've taken this logic from `OrganizationEventsEndpointTestBase`. Only typing information has been added.